package.json

```diff
@@ -1,28 +1,23 @@
 {
   "name": "@replit/extensions",
-  "version": "0.32.1",
+  "version": "0.33.0",
   "description": "",
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "types": "./dist/index.d.ts",
-      "import": {
-        "default": "./dist/index.js"
-      },
-      "default": "./dist/index.cjs"
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs",
+      "types": "./dist"
     },
     "./react": {
-      "types": "./dist/react/index.d.ts",
-      "import": {
-        "types": "./dist/react/index.d.ts",
-        "default": "./dist/react/index.js"
-      },
-      "default": "./dist/react/index.cjs"
+      "import": "./dist/react/index.js",
+      "require": "./dist/react/index.cjs",
+      "types": "./dist/react"
```

Comment:
have we made sure both `require` and es6 `import` works?

Reply:
I remember arriving at that structure after a lot of iteration so that every type of import works

Reply:
I did a lot of research, and that is how you do it.  I haven't tested it out, but theoretically it should work.

If we need to test it out, how would we do that?

Reply:
Added a `default` export to the list, nothing should be missed now